CSS properties: intro sentence in values section part 3 (#44936)
mdn/content: files/en-us/web/css/reference/properties/dynamic-range-limit/index.md

### Values

> This property is specified as one of the following keyword values or a `dynamic-range-limit-mix()` function:

Reviewer: ```suggestion
This property is specified as a single value from the following list:
```
